packages/shared/src/components/post/PostContentHeader.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import { useRouter } from 'next/router';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { ArrowIcon } from '../icons';
+import { PostHeaderActions } from './PostHeaderActions';
+import { PostNavigationProps } from './common';
+import { SourceType } from '../../graphql/sources';
+
+export function PostContentHeader({
+  post,
+  onReadArticle,
+}: Pick<PostNavigationProps, 'onReadArticle' | 'post'>): ReactElement {
+  const router = useRouter();
+  const canGoBack = !!globalThis?.window?.history?.length;
+  const routedFromSquad = router?.query?.squad;
```

Comment:
Before we also had from notification right? (not sure we need it, just raising it)

Reply:
And button will no longer have copy? (eg back or back to squad?)

Reply:
Going back to the notifications should be automatic since it will be a part of the history.

Yeah, no more copy, as per design. I think the reason is that "back to squad" is a bit long and might interfere with the action buttons.